torch/testing/_internal/common_optimizers.py

```diff
@@ -545,6 +544,21 @@ def optim_error_inputs_func_lbfgs(device, dtype):
 def optim_inputs_func_nadam(device=None):
     cuda_supported_configs = [
         OptimizerInput(params=None, kwargs={"capturable": True}, desc="capturable"),
+        OptimizerInput(
```

Comment:
Add more configs for capturable for coverage--this _technically_ doesn't have to be in this PR...but if CI passes, then why not?